assets/models/system/automl-ner/spec.yaml

```diff
@@ -35,61 +34,58 @@ tags:
       Standard_NC16as_T4_v3,
       Standard_NC24s_v3,
       Standard_NC64as_T4_v3,
-      Standard_NC24ads_A100_v4,
-      Standard_NC48ads_A100_v4,
       Standard_NC96ads_A100_v4,
       Standard_ND96asr_v4,
       Standard_ND96amsr_A100_v4,
       Standard_ND40rs_v2
     ]
   inference_compute_allow_list:
     [
-        Standard_D4a_v4,
-        Standard_D4as_v4,
-        Standard_DS4_v2,
-        Standard_D8a_v4,
-        Standard_D8as_v4,
-        Standard_DS5_v2,
-        Standard_D16a_v4,
-        Standard_D16as_v4,
-        Standard_D32a_v4,
-        Standard_D32as_v4,
-        Standard_D48a_v4,
-        Standard_D48as_v4,
-        Standard_D64a_v4,
-        Standard_D64as_v4,
-        Standard_D96a_v4,
-        Standard_D96as_v4,
-        Standard_F4s_v2,
-        Standard_FX4mds,
-        Standard_F8s_v2,
-        Standard_FX12mds,
-        Standard_F16s_v2,
-        Standard_F32s_v2,
-        Standard_F48s_v2,
-        Standard_F64s_v2,
-        Standard_F72s_v2,
-        Standard_FX24mds,
-        Standard_FX36mds,
-        Standard_FX48mds,
-        Standard_E4s_v3,
-        Standard_E8s_v3,
-        Standard_E16s_v3,
-        Standard_E32s_v3,
-        Standard_E48s_v3,
-        Standard_E64s_v3,
-        Standard_NC4as_T4_v3,
-        Standard_NC6s_v3,
-        Standard_NC8as_T4_v3,
-        Standard_NC12s_v3,
-        Standard_NC16as_T4_v3,
-        Standard_NC24s_v3,
-        Standard_NC64as_T4_v3,
-        Standard_NC24ads_A100_v4,
-        Standard_NC48ads_A100_v4,
-        Standard_NC96ads_A100_v4,
-        Standard_ND96asr_v4,
-        Standard_ND96amsr_A100_v4,
-        Standard_ND40rs_v2
+      Standard_D4a_v4,
+      Standard_D4as_v4,
+      Standard_DS4_v2,
+      Standard_D8a_v4,
+      Standard_D8as_v4,
+      Standard_DS5_v2,
+      Standard_D16a_v4,
+      Standard_D16as_v4,
+      Standard_D32a_v4,
+      Standard_D32as_v4,
+      Standard_D48a_v4,
+      Standard_D48as_v4,
+      Standard_D64a_v4,
+      Standard_D64as_v4,
+      Standard_D96a_v4,
+      Standard_D96as_v4,
+      Standard_FX4mds,
+      Standard_F8s_v2,
+      Standard_FX12mds,
+      Standard_F16s_v2,
+      Standard_F32s_v2,
+      Standard_F48s_v2,
+      Standard_F64s_v2,
+      Standard_F72s_v2,
+      Standard_FX24mds,
+      Standard_FX36mds,
+      Standard_FX48mds,
+      Standard_E4s_v3,
+      Standard_E8s_v3,
+      Standard_E16s_v3,
+      Standard_E32s_v3,
+      Standard_E48s_v3,
+      Standard_E64s_v3,
+      Standard_NC4as_T4_v3,
+      Standard_NC6s_v3,
+      Standard_NC8as_T4_v3,
+      Standard_NC12s_v3,
+      Standard_NC16as_T4_v3,
+      Standard_NC24s_v3,
+      Standard_NC64as_T4_v3,
+      Standard_NC24ads_A100_v4,
+      Standard_NC48ads_A100_v4,
+      Standard_NC96ads_A100_v4,
+      Standard_ND96asr_v4,
+      Standard_ND96amsr_A100_v4,
+      Standard_ND40rs_v2
     ]
-version: 1
+version: 2
```

Comment:
we can either increment version of all models with property changes in this PR or do it separately to make sure we dont increment model version more than once

Reply:
Let's do it in this PR itself. And, not o it later.